<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KadasGlobeDialogBase</class>
 <widget class="QDialog" name="KadasGlobeDialogBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>534</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Globe Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMap">
      <attribute name="title">
       <string>Map</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBoxSky">
         <property name="title">
          <string>Sk&amp;y</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxDateTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Override Date / Time (UTC)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateTimeEdit" name="dateTimeEditSky">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy HH:mm</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxSkyAutoAmbient">
            <property name="text">
             <string>Ambient lighting</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="horizontalSliderMinAmbient">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="mImageryGroupBox">
         <property name="title">
          <string>Imagery</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTreeWidget" name="mImageryTreeView">
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>50</number>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="mAddImageryButton">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mRemoveImageryButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacerMapTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="mElevationGroupBox">
         <property name="title">
          <string>Elevation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QLabel" name="labelVerticalScale">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Vertical scale</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="mSpinBoxVerticalScale">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTreeWidget" name="mElevationTreeView">
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>50</number>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="mAddElevationButton">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mRemoveElevationButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVideo">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxAntiAliasing">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Anti Aliasing</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelAASamples">
            <property name="text">
             <string>Samples</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditAASamples">
            <property name="placeholderText">
             <string>[Leave empty for maximum]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelVideoInfo">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Change requires a restart of the globe plugin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacerVideoTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Stereo</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="labelStereoMode">
            <property name="text">
             <string>Stereo Mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxStereoMode"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelEyeSeparation">
            <property name="text">
             <string>Eye separation (m)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxStereoEyeSeparation">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.001000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelScreenDistance">
            <property name="text">
             <string>Screen distance (m)</string>
            </property>
            <property name="buddy">
             <cstring>spinBoxStereoScreenDistance</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxStereoScreenDistance">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelScreenWidth">
            <property name="text">
             <string>Screen width (m)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxStereoScreenWidth">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelScreenHeight">
            <property name="text">
             <string>Screen height (m)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxStereoScreenHeight">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelSplitStereoHorizontalSeparation">
            <property name="text">
             <string>Split stereo horizontal separation (px)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="spinBoxSplitStereoHorizontalSeparation">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelSplitStereoHorizontalEyeMapping">
            <property name="text">
             <string>Split stereo horizontal eye mapping</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="comboBoxSplitStereoHorizontalEyeMapping">
            <item>
             <property name="text">
              <string notr="true">LEFT_EYE_LEFT_VIEWPORT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">LEFT_EYE_RIGHT_VIEWPORT</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelSplitStereoVerticalSeparation">
            <property name="text">
             <string>Split stereo vertical separation (px)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="spinBoxSplitStereoVerticalSeparation">
            <property name="accelerated">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelBoxSplitStereoVerticalEyeMapping">
            <property name="text">
             <string>Split stereo vertical eye mapping</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="comboBoxSplitStereoVerticalEyeMapping">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string notr="true">LEFT_EYE_TOP_VIEWPORT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">LEFT_EYE_BOTTOM_VIEWPORT</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pushButtonStereoResetDefaults">
            <property name="text">
             <string>Reset to defaults</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxScrolling">
         <property name="title">
          <string>Scrolling</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="2" column="0">
           <widget class="QLabel" name="labelScrollSensitivity">
            <property name="text">
             <string>Sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="sliderScrollSensitivity">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxInvertScroll">
            <property name="text">
             <string>Invert scroll wheel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxFrustumHighlighting">
         <property name="text">
          <string>Enable frustum highlighting</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerAdvancedTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
